addons/main/XEH_postInit.sqf

```diff
@@ -515,4 +515,6 @@ GVAR(addingPlate) = false;
 },
 [DIK_T, [false, false, false]], false] call CBA_fnc_addKeybind;
 
+player call FUNC(regenBleedout)
```

Comment:
Needs to check if it's enabled. The setting needs to be marked as needing restart if this is only called from postInit.
It could be possibly be moved to an event handler, but then would need a handle for checking if it's already active.

Reply:
That's probably the smart way to do it. Will do shortly. 
The way I set it up is (intended) to have it always run, but do nothing if not enabled, to not require a restart to toggle it. But toggling it on-line isn't really important. Changing the coefficient should still be dynamic, right?

Reply:
Agreed, as the function it self is lacking a stop. Either add a check here, or one at the top of the function itself.